.agents/structure.md

```
@@ -4,33 +4,43 @@
+│   │   └── service-worker.ts # Service Worker entry
+│   ├── dispatcher/         # Dispatcher interface and pipeline triggers
+│   ├── event/              # RoutupEvent class and IRoutupEvent interface
 │   ├── error/              # RoutupError (extends @ebec/http)
 │   ├── handler/            # Handler definitions
 │   │   ├── core/           # coreHandler — standard request handlers
-│   │   └── error/          # errorHandler — error-specific handlers
+│   │   ├── error/          # errorHandler — error-specific handlers
+│   │   └── helpers/        # fromNodeHandler — Node.js middleware bridge
 │   ├── hook/               # Lifecycle hook system
 │   ├── path/               # PathMatcher using path-to-regexp
 │   ├── plugin/             # Plugin install/management
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)